cfme/common/provider.py

```diff
@@ -1063,6 +1063,22 @@ def num_vm(self):
     def num_vm_ui(self):
         return int(self.get_detail("Relationships", self.vm_name))
 
+    @variable(alias="db")
+    def num_availability_zone(self):
+        pass  # TODO: come up with a db query
+
+    @num_availability_zone.variant('ui')
+    def num_availability_zone_ui(self):
+        return int(self.get_detail("Relationships", "Availability Zones"))
+
+    @variable(alias="db")
+    def num_orchestration_stack(self):
+        pass  # TODO: come up with a db query
+
+    @num_orchestration_stack.variant('ui')
+    def num_orchestration_stack_ui(self):
+        return int(self.get_detail("Relationships", "Orchestration Stacks"))
```

Comment:
**QUESTION**
Are these properties common for all CloudInfraProviders subclasess? If not please put them into respectful class.